subs/build_miner.sh

```diff
@@ -66,6 +66,11 @@ init_update_external_modules
 autoreconf -i
 
 ./configure --prefix=$BUILD_DIR/output CFLAGS="-I$BUILD_DIR/output/include" LDFLAGS="-L$BUILD_DIR/output/lib" CPPFLAGS="-I$BUILD_DIR/output/include" LIBCURL_CFLAGS="-I$BUILD_DIR/output/include/curl" LIBCURL_LIBS="-L$BUILD_DIR/output/lib -lcurl" LIBUSB_CFLAGS="-I$BUILD_DIR/output/include/libusb-1.0" LIBUSB_LIBS="-L$BUILD_DIR/output/lib -lusb-1.0" --enable-scrypt --enable-keccak --enable-bflsc --enable-bitforce --enable-icarus --enable-modminer --enable-ztex --enable-avalon --disable-adl --enable-opencl --enable-bitfury --enable-hashfast --enable-klondike --enable-cointerra --enable-blockerupter --enable-hashratio --enable-avalon2 --enable-drillbit
+
+if [ $MINER_EXE == 'cgminer' ] && [ $MINER_VERSION == $CGMINER_VERSION ]; then
+cp $BUILD_DIR/output/include/libusb-1.0/libusb.h $BUILD_DIR/working/libusb.h
+fi
+
```

Comment:
This should not be necessary at all. It seems like the cgminer `configure` script is not picking up `LIBUSB_CFLAGS` which is passed in to `configure`.

Instead, please take care of this by passing the include path for libusb in `CFLAGS`:

before:
```
./configure \
    --prefix=$BUILD_DIR/output \
    CFLAGS="-I$BUILD_DIR/output/include" \
    LDFLAGS="-L$BUILD_DIR/output/lib" \
    CPPFLAGS="-I$BUILD_DIR/output/include" \
    LIBCURL_CFLAGS="-I$BUILD_DIR/output/include/curl" \
    LIBCURL_LIBS="-L$BUILD_DIR/output/lib -lcurl" \
    LIBUSB_CFLAGS="-I$BUILD_DIR/output/include/libusb-1.0" \
    LIBUSB_LIBS="-L$BUILD_DIR/output/lib -lusb-1.0" \
```

after:
```
./configure \
    --prefix=$BUILD_DIR/output \
    CFLAGS="-I$BUILD_DIR/output/include -I$BUILD_DIR/output/include/libusb-1.0" \
    LDFLAGS="-L$BUILD_DIR/output/lib" \
    CPPFLAGS="-I$BUILD_DIR/output/include" \
    LIBCURL_CFLAGS="-I$BUILD_DIR/output/include/curl" \
    LIBCURL_LIBS="-L$BUILD_DIR/output/lib -lcurl" \
    LIBUSB_CFLAGS="-I$BUILD_DIR/output/include/libusb-1.0" \
    LIBUSB_LIBS="-L$BUILD_DIR/output/lib -lusb-1.0" \
```

Reply:
I attempted this, but I still get the error
```
Package libusb-1.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `libusb-1.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'libusb-1.0' found
In file included from cgminer.c:58:
In file included from ./sha2.h:36:
./miner.h:122:12: fatal error: 'libusb.h' file not found
  #include <libusb.h>
           ^~~~~~~~~~
1 error generated.
make[2]: *** [cgminer-cgminer.o] Error 1
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```
I do however have an unlinked homebrew libusb on my system, I'll give it a shot in another environment

Reply:
This worked on a clean system so please disregard the error